blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -62,7 +62,8 @@ export class RunDetailsPipeline extends Component {
             }
         }
 
-        this.listener.sse = sseConnection.subscribe('pipeline', this._onSseEvent);
+        this.listener.ssePipeline = sseConnection.subscribe('pipeline', this._onSseEvent);
+        this.listener.sseJob = sseConnection.subscribe('job', this._onSseEvent);
```

Comment:
hmm, the listener is an Object `{}` that subscribe to two different channels and then stores them. Not sure how we could do it differently.